ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -277,19 +289,41 @@
           </label>
           <v-row>
             <v-col col="12">
+              <v-tooltip
+                v-if="owner.partyType === HomeOwnerPartyTypes.EXECUTOR"
+                right
+                content-class="right-tooltip pa-5"
+                transition="fade-transition"
+              >
+                <template v-slot:activator="{ on }">
+                  <v-text-field
+                    v-on="on"
+                    id="suffix"
+                    v-model="owner.suffix"
+                    filled
+                    label="Additional Name Information (Optional)"
+                    data-test-id="suffix"
+                    hint="Example: Additional legal names, Jr., Sr., Executor of the will of the deceased, etc."
+                    persistent-hint
+                    :rules="maxLength(70)"
+                    @mousedown.prevent
+                  />
+                </template>
+                  Executor of the will is based on deceased owner with Grant of Probate
+                  with Will supporting document selected.
+              </v-tooltip>
               <v-text-field
-                id="suffix"
-                v-model="owner.suffix"
-                filled
-                label="Additional Name Information (Optional)"
-                data-test-id="suffix"
-                hint="Example: Additional legal names, Jr., Sr., Executor of the will of the deceased, etc."
-                persistent-hint
-                :rules="maxLength(70)"
-                :disabled="isTransferDueToDeath"
-                :readonly="isTransferDueToDeath"
-                @mousedown.prevent
-              />
+                  v-else
+                  id="suffix"
+                  v-model="owner.suffix"
+                  filled
+                  label="Additional Name Information (Optional)"
+                  data-test-id="suffix"
+                  hint="Example: Additional legal names, Jr., Sr., Executor of the will of the deceased, etc."
+                  persistent-hint
+                  :rules="maxLength(70)"
+                  @mousedown.prevent
+                />
```

Comment:
Few things, these Text fields look identical no? Can't we disable the tooltip opposed to duplicating the text field? 

Also, the disabled and readonly props are class activators for the 'special' disabled state the designers are going for. 
For example, in a TRAND filing, these will be disabled/readonly, the classes assigned by these props are required in order to override the default vuetify state. 

Hard to describe the design requirements, but essentially no mouse activations/highlighting and so on.

Reply:
Made that change. The thought crossed my mind but I assumed disabling the tooltip would disable the whole text field because its templated within the tooltip. Learnt that's not the case tho!